README.md

```diff
@@ -38,6 +38,11 @@ django-admin-interface is a modern **responsive flat admin interface customizabl
 - Run `pip install django-admin-interface`
 - Add `admin_interface`, `flat_responsive`, `flat` and `colorfield` to `settings.INSTALLED_APPS` **before** `django.contrib.admin`
 ```python
+
+import os                                          // you need to import this package to run the following command.
+STATIC_DIR = os.path.join(BASE_DIR,'static')         // Before running this command, you must create your static directory under your project directory. This variable will have                                                            a path of the static folder, which you have created for your project.
+STATIC_ROOT = STATIC_DIR                            // This variable will be required to store all the static files of our library into your static directory.
```

Comment:
Hello!  some comments:

- the comments are not valid Python, and very long
- why define STATIC_DIR?  Django only looks at STATIC_ROOT
- I suggest the doc uses `directory` consistently, not `folder`

Reply:
Thanks to @merwok 
I was just trying to contribute to the open-source project.
I am just a beginner.
Would appreciate it if you suggest something for improving my skills in open source.